CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,157 @@
+# Zarf Contributor Ladder
+
+This document outlines the various contributor roles in the Zarf project, along with their respective prerequisites and responsibilities.
+It also defines the process by which users can request to change roles.  These roles are progressive, in that responsibilites and pre-requisites apply to subsequent rungs in the ladder.
+
+- [Roles](#roles)
+  - [Community Participants](#community-participants)
+  - [Contributors](#contributors)
+  - [Reviewers](#reviewers)
+  - [Maintainers](#maintainers)
+- [Inactive members](#inactive-members)
+
+## Roles
+
+### Community participants
+
+Community participants engage with Zarf, 
+contributing their time and energy in discussions or just generally helping out.  Additionally, community members participate in [Zarf community meetings](https://github.com/zarf-dev/zarf/issues/2613).
+
+#### Responsibilities
+
+- Keep participating!
+
+#### Prerequisites
+
+- Must follow the [OpenSSF Code of Conduct]
+- Must follow the [Contribution Guide]
+
+### Contributors
+
+Contributors help advance the Zarf project through commits, issues, and pull requests.  Contributors participate through GitHub teams,
+and pre-submit tests are automatically run for their PRs.
+
+#### Responsibilities
+- Can be assigned issues and PRs
+- Responsive to issues and PRs assigned to them
+- Others can ask for reviews with a `/cc @username`.
+- Responsive to mentions of teams they are members of
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+  - Ensures code is well tested and that tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+
+#### Privileges
+
+- Tests run against their PRs automatically
+
+#### Prerequisites
+
+- Enabled two-factor authentication on their GitHub account
+- Have made contributions to the project in the form of:
+  - Authoring or reviewing PRs on GitHub. At least one PR must be **merged**.
+  - Filing or commenting on issues on GitHub
+  - Contributing to a project, or community discussions (e.g. meetings, Slack,
+    email discussion forums, Stack Overflow)
+- Active contributor to Zarf
+
+#### Promotion process `NEEDS REFINED?`
+
+- Make at least one commit to a repository's code or open a pull request that gets merged into the repository
+
+### Reviewers
+
+Reviewers are trusted members of the Zarf community that are able to review changes to Zarf and indicate if those changes are ready for merge.  Reviewers have a strong and active track record of contribution to the Zarf project.
+
+#### Responsibilities
+
+Commits to being an active contributor and reviewer as part of the Zarf project.  'Active' is defined as six PRs or PR reviews (or mix thereof) in six months.
```

Comment:
I like the "active" metric - we should ensure these are able to be queried if we intend to validate them.